configuration/init/systemd/tedge-mapper-az@.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=tedge-mapper-az checks Thin Edge JSON measurements and forwards to Azure IoT Hub.
 After=syslog.target network.target mosquitto.service
+PartOf=tedge-mapper-aws.target
```

Comment:
```suggestion
PartOf=tedge-mapper-az.target
```

Reply:
Resolved in ac91b0192809bc0880a763d83f6c87810d34eedd